_layouts/docu.html

```diff
@@ -85,7 +85,10 @@
 				<div style="width: 0.2em"></div>
 
 				<div class="improvethispage">
-					<a href="https://github.com/duckdb/duckdb-web/issues/new?title={{ "Issue found on page '" | append: page.title | append: "'" | uri_escape  }}&body={{ "Page URL: <https://duckdb.org/" | append: page.url | append: ">" | uri_escape }}" title="Create GitHub issue">Report Issue</a>
+					<a href="https://github.com/duckdb/duckdb-web/issues/new?title={{ "Issue found on page '" | append: page.title | append: "'" | uri_escape  }}&body={{ "
+
+
+Page URL: <https://duckdb.org" | append: page.url | append: ">" | uri_escape }}" title="Create GitHub issue">Report Issue</a>
```

Comment:
Maybe build this up incrementally using assign instead?

Reply:
Thx, fixed.